src/ui/ui.css

```diff
@@ -20,17 +20,44 @@
   position: relative;
 }
 
+/*Formula bar*/
+.light_sheet_table_formula_bar {
```

Comment:
for css class names use the prefix as lightsheet everywhere instead of light_sheet